### PR TITLE
[FLINK-12421][docs-zh] Synchronize the latest documentation changes (commits to 7b46f604) into Chinese documents

### PR DESCRIPTION
**Pull request by wuchong**
## What is the purpose of the change

There are several commits to documentations have not been synchronized to Chinese documents, i.e. `xx.zh.md`. This pull request will synchronize the latest changes into Chinese documents.

## Brief change log

  - Synchronize updates to `xx.zh.md`
  - Fix the `Overview` is not translated in sidebar

## Verifying this change

N/A

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by StephanEwen** [comment]
Looks good.

+1 to merge this

File: docs/dev/connectors/kafka.zh.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ For most users, the `FlinkKafkaConsumer08` (part of `flink-connector-kafka`) is
         <td>>= 1.0.0</td>
         <td>
         This universal Kafka connector attempts to track the latest version of the Kafka client.
-        The version of the client it uses may change between Flink releases.
+        The version of the client it uses may change between Flink releases. As of this release, it uses the Kafka 2.2.0 client.
```
Comment (fhueske) [inline docs/dev/connectors/kafka.zh.md]: I think we should not reference `As of this release`. In 6 months nobody knows what this refers to.
Could you replace that with the correct version (presumably 1.8) in both languages?

Comment (wuchong) [reply docs/dev/connectors/kafka.zh.md]: This sentence was introduced in FLINK-12024 which affects only 1.9. 

What about change this to `Starting with Flink 1.9 release, it uses...` ?

Comment (fhueske) [reply docs/dev/connectors/kafka.zh.md]: Perfect. Thank you!

**Comment by wuchong** [comment]
Thanks @fhueske , I will merge then.

**Pull request closed without merging**